apps/finance/app/src/components/BalanceToken.js

```diff
@@ -63,9 +69,21 @@ const BalanceToken = ({
           css={`
             ${textStyle('title2')}
             margin: ${(compact ? 1 : 1.5) * GU}px 0;
+            display:flex;
           `}
         >
-          {splitAmount(amount.toFixed(3))}
+          {splitAmount(integerAmount, fractionalAmount)}
+          {amountWasRounded && (
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+                margin-left: ${GU}px;
+              `}
+            >
+              <Help>{amount}</Help>
```

Comment:
We should also add a [hint](https://github.com/aragon/aragon-ui/blob/master/src/components/Help/Help.js#L67) on `Help`. Perhaps something like “This is an approximation, see the complete amount”? This is what gets used as a `title` attribute for the button. /cc @dizzypaty @sohkai

Reply:
> “This is an approximation, see the complete amount”

Agreed!